index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications to consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if present, using the <code>@id</code> JSON key, which
+is aliased in this specification to <code>id</code>. See
```

Comment:
I wonder whether for the prospective readers of this specification, mentioning `@id` may do more harm than good.
 
```suggestion
identifier of a subject using, if necessary, the <code>id</code> key in this specification. See
```